doc/python-usage.rst

```diff
@@ -354,6 +354,31 @@ a translation mapping is required, as follows:
 
 This should animate a green dot moving left-to-right down each line.
 
+Pimoroni Unicorn HAT HD
+"""""""""""""""""""""""
+Pimoroni sells the `Unicorn HAT HD <https://shop.pimoroni.com/products/unicorn-hat-hd>`_, 
+comprising 256 high-intensity RGB LEDs a 16x16 arrangement. The pixels driven by ARM STM32F
+making the display appear as an SPI device:
+
+.. code:: python
+
+    import time
```

Comment:
would be better to use monotonic time here but, since it's not in Py27, it would overcomplicate the example

Reply:
\*nod\*